src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,17 @@
+@mixin circle ($diametr) {
+  width: $diametr;
+  height: $diametr;
+  border-radius: 50%;
+}
+
+@mixin minutes-hand-size ($width, $height) {
+  width: $width;
+  height: $height;
+  border-radius: 50% 50% 0 0;
+}
+
+@mixin seconds-hand-size ($width, $height) {
+  width: $width;
+  height: $height;
+  border-radius: 50% 50% 0 0;
+}
```

Comment:
the same code, u can use


`@mixin minutes-hand-size ($width, $height),
@mixin seconds-hand-size ($width, $height){
}
` instead